src/utilities/helpers.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * @description Checks whether provided string is a valid URL.
+ * @param {string} string
+ * @return {boolean} Whether the string is a valid URL.
+ */
+
+export const isValidUrl = (string: string): boolean => {
+  try {
+    // eslint-disable-next-line no-new
+    new URL(string)
+
+    // No error was thrown, so return true
+    return true
+  } catch {
+    // Invalid URL
+    return false
+  }
+}
```

Comment:
nit: can you rename this file to `urls` or something? We currently have each utility file delegated to a single type of "thing" that it does

Reply:
Done